.github/workflows/test.yml

```diff
@@ -17,25 +31,6 @@ env:
   PYTEST_ADDOPTS: "--verbose --color=yes"
 
 jobs:
-  rest-api:
```

Comment:
Only by extracting this job to a dedicated workflow file could we avoid triggering the workflow with all tests on docs changes.